### PR TITLE
feat(plugins) introduce new plugin properties for kong-bundle

### PR DESCRIPTION
**Pull request by nijikokun**
Introduces new [`package.json`](https://github.com/Kong/insomnia/blob/19f449ba0286eee7c015dd29db70812e6bc193fa/plugins/insomnia-plugin-kong-bundle/package.json#L5) properties for upcoming plugin hub updates to `insomnia-plugin-kong-bundle`.

- Add `$.insomnia.displayName` (String) for plugin display name on UI & GUIs
- Add `$.insomnia.bundle` (Boolean) boolean flag denotes bundle status to show individual plugins on hub
- Add `$.insomnia.applications` (Object) object to be used for install buttons and checks
- Add `$.insomnia.images.icon` (String, Relative Path) property for plugin icon
- Add `$.insomnia.images.cover` (String, Relative Path) property for plugin cover image

Other Properties that can be used but not added:

- `$.insomnia.publisher.name` (String) to denote Plugin Author name on GUI
- `$.insomnia.publisher.icon` (String, URL) to denote Plugin Author icon

**Visual Elements:**

Icon

Cover Image

### CODE REVIEW

File: plugins/insomnia-plugin-kong-bundle/package.json [hunk 1]
```diff
@@ -4,11 +4,20 @@
   "main": "index.js",
   "insomnia": {
     "name": "kong-bundle",
-    "description": "Bundle of all Kong plugins"
+    "displayName": "Kong Plugin Bundle",
+    "description": "Bundle of all Kong functionality",
+    "bundle": true,
+    "applications": {
+      "designer": "*"
```
Comment (develohpanda) [inline plugins/insomnia-plugin-kong-bundle/package.json:11]: - [x] Is the value (`*`) to denote supported versions of the application?
- [ ] Should there be a check in the application during installation, to ensure the _running application and version_ are supported by the plugin?

**Review by develohpanda — APPROVED** [review]
LGTM, non-blocking questions 👍 

Lovin' the cover image - should the `applications` and `images` fields also be added to the individual `kong-*` plugins (given they are also visible on the plugin hub individually)?

File: plugins/insomnia-plugin-kong-bundle/package.json (hunk 1 above)
Comment (nijikokun) [reply plugins/insomnia-plugin-kong-bundle/package.json:11]: To the second item, yes. Can make an issue that references this.

**Comment by nijikokun** [comment]
> LGTM, non-blocking questions 👍
❤️ 

> Lovin' the cover image - should the `applications` and `images` fields also be added to the individual `kong-*` plugins (given they are also visible on the plugin hub individually)?

Yes, will do it in subsequent PRs, this is just so I can get started on the plugin hub features

**Pull request merged**